src/pages/contact/contact-form-section.tsx

```diff
@@ -30,9 +20,34 @@ class ContactFormSection extends React.Component<IProps, IState> {
     };
   }
 
+  formValid = () => {
+    let formErrors = this.state.formErrors;
+    let name = this.state.name;
+    let email = this.state.email;
+    let message = this.state.message;
+    let valid = true;
+
+    if(name && name.length < 10){
```

Comment:
No need to set a length for name. It just a required field. 
`if (name && name.length === 0)` **or**
`if (name)`
Both are correct way to check empty field.

Reply:
Please consider above request